homeassistant/helpers/script.py

```diff
@@ -574,8 +580,13 @@ async def _async_device_step(self):
         platform = await device_automation.async_get_device_automation_platform(
             self._hass, self._action[CONF_DOMAIN], "action"
         )
+
+        rendered_action = template.render_complex(
```

Comment:
oh, this isn't doing the platform-specific config at this point -- same issue as above, it's gonna be upset about being run twice on the same set of (base) values